databox-build-core

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+RED='\033[0;31m'
+GREEN='\033[0;32m' 
+NC='\033[0m'
+ME='databox-build-core'
+
+function fail {
+    echo -e "[$(date +'%Y-%m-%dT%H:%M:%S%z') $ME]: ${1} \nERROR: ${2} ${RED}FAILED${NC}"
+    exit 1
+}
+
+function success {
+    echo -e "[$(date +'%Y-%m-%dT%H:%M:%S%z') $ME]: ${1} ${GREEN}OK${NC}"
+}
+
+function test_assert {
+  if [ "$1" != "$2" ]
+  then
+    fail "$3" "$4"
+  else
+    success "$3"
+  fi
+}
+
+build() 
+{
+    echo "[$(date +'%Y-%m-%dT%H:%M:%S%z') $ME]: Starting build ${1} ..."
```

Comment:
Too many repeats of that very long `date ...` invocation. Refactor to shell functions that format the line appropriately?

Reply:
done